packages/portal/src/pages/galleries/_.vue

```diff
@@ -125,105 +125,47 @@
           />
         </div>
       </b-container>
-      <LoadingSpinner
```

Comment:
Don't we want to keep the loading spinner here?

Reply:
It's included earlier on in the component.